osu.Game/Rulesets/Edit/Checks/CheckTooShortAudioFiles.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using ManagedBass;
+using osu.Framework.Audio.Callbacks;
+using osu.Game.Rulesets.Edit.Checks.Components;
+
+namespace osu.Game.Rulesets.Edit.Checks
+{
+    public class CheckTooShortAudioFiles : ICheck
+    {
+        private const int ms_threshold = 25;
+        private const int min_bytes_threshold = 100;
+
+        private readonly string[] audioExtensions = { "mp3", "ogg", "wav" };
+
+        public CheckMetadata Metadata => new CheckMetadata(CheckCategory.Audio, "Too short audio files");
```

Comment:
```suggestion
        public CheckMetadata Metadata => new CheckMetadata(CheckCategory.Audio, "Very short audio files");
```

I think this reads better. The class name can probably stay as is, not sure.

Reply:
Pattern I've followed so far is using "too" when it's a problem and "very" when it's a warning - it's all relative to what the Ranking Criteria allows is the idea.

Reply:
Can it be changed to "Audio files too short" maybe? I don't know the technical term for what feels wrong about the "sentence" structure you currently have, but something is off.

Reply:
Does this even work in all languages? Imo it's best to have natural english and leave it up to other displays to tell you whether something is a problem or a warning.

Reply:
Hmm it sounds pretty natural to me, and I'm quite sure most languages have different terms for "too much" vs "a lot". I don't see "Audio files too short" as any better or worse than it currently is, so if you think that sounds better then sure.

Reply:
Should I change the other check names too, given that they follow the same structure? Would make the most sense to keep things consistent, I think.

"Too high or low audio bitrate" -> "Audio bitrate too high or low"
"Unsnapped hitobjects" -> "Hitobjects unsnapped"
"Missing or unexpected overlaps" -> "Overlaps missing or unexpected"
etc